src/Telemetry/Core.php

```diff
@@ -123,7 +125,9 @@ public function container() {
 	private function init_container( string $plugin_path ): void {
 		$container = Config::get_container();
 
+		$container->bind( self::PLUGIN_SLUG, dirname( plugin_basename( $plugin_path ) ) );
 		$container->bind( self::PLUGIN_BASENAME, plugin_basename( $plugin_path ) );
+		$container->bind( self::PLUGIN_DATA, get_plugin_data( $plugin_path ) );
```

Comment:
❓ 
Wouldn't this only provide the plugin data for the current instance of the library and not all the plugins registered?

Reply:
yes, but each instance of the library is responsible for adding its own plugin to the `stellarwp_telemetry` option. Once the plugin is added to that option, we have everything we need.
https://github.com/stellarwp/telemetry/blob/55fe91e8c7668d064df00248504e0793e1b163f6/src/Telemetry/Opt_In/Status.php#L144-L155

Reply:
that makes sense.